front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/index.ts

```diff
@@ -95,6 +95,9 @@ async function handler(
           topK: action.topK,
           dataSources: action.dataSources,
         });
+      } else {
+        // in this case I think we want to to delete the action configuration
```

Comment:
We need to keep the action configuration as it is potentially linked from older actions but we want to unlink it from the agent configuration indeed :+1: